datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/analytics/service/AnalyticsUtil.java

```diff
@@ -151,6 +159,60 @@ public static void hydrateDisplayNameForTable(
                     }));
   }
 
+  public static void convertToUserInfoRows(
+      OperationContext opContext, EntityClient entityClient, List<Row> rows) throws Exception {
+    final Set<Urn> userUrns =
+        rows.stream()
+            .filter(row -> !row.getCells().isEmpty())
+            .map(row -> UrnUtils.getUrn(row.getCells().get(0).getValue()))
+            .collect(Collectors.toSet());
+    final Map<Urn, EntityResponse> gmsResponseByUser =
```

Comment:
Might be a good idea to add a check for userUrns size before making these calls.